spec/controllers/people_controller_spec.rb

```diff
@@ -57,8 +57,8 @@
     describe 'via html' do
       context 'query is a diaspora ID' do
         before do
-          @unsearchable_eugene = FactoryGirl.create(:person, :diaspora_handle => "eugene@example.org",
-                                         :profile => FactoryGirl.build(:profile, :first_name => "Eugene",
+          @unsearchable_eugene = FactoryBot.create(:person, :diaspora_handle => "eugene@example.org",
+                                         :profile => FactoryBot.build(:profile, :first_name => "Eugene",
```

Comment:
- Layout/HashAlignment: Align the keys and values of a hash literal if they span more than one line.
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.